firestore/google/cloud/firestore_v1/client.py

```diff
@@ -23,6 +23,7 @@
 * a :class:`~google.cloud.firestore_v1.client.Client` owns a
   :class:`~google.cloud.firestore_v1.document.DocumentReference`
 """
+from google.api_core import grpc_helpers
```

Comment:
This module isn't generated code.

Reply:
Correct. Is there a reason the change seems incorrect for manual code?

Reply:
Your comment above says:

> These changes are in generated code here though and shouldn't be merged.

I was pointing out that none of this PR touches generated code.

Reply:
Ah. Thanks. You are correct. I originally did this in generated code and then made modifications